Sources/KeyboardShortcuts/Recorder.swift

```diff
@@ -31,39 +31,70 @@ extension KeyboardShortcuts {
 
 		private let name: Name
 		private let onChange: ((_ shortcut: Shortcut?) -> Void)?
+        private let isHorizontalContentSizeConstraintActive: Bool
```

Comment:
Even if we decide to introduce a new option, this option is too specific.

Reply:
Any counter proposal? The actual init parameter is called `constraint`.

Reply:
We might go this approach: We completely get right of the built-in width constraint. The control is utilizing its width to the available space completely. 

Seeing how all other native SwiftUI controls are implemented, I would say that this behavior would be more SwiftUI'ful, because their widths is always controlled through either modifiers (e.g. `.frame()`), or ViewBuilder structs.
I haven't seen any native SwiftUI control that has a fixed width constraint.

What do you think?

Reply:
I partly agree. The control still needs a minimum width (and intrinsic content size) to make sure the label is not cut off.

Reply:
OK, I see your point. Nevertheless, this is a topic every UI designer has to handle with each and every other native controls such as password textfield etc. 

I see your point that the keyboard recorder control requires a minimum width.
Maybe we could control the minimum width in the `override public var intrinsicContentSize: CGSize { ... }` function by not allowing a width of the size below a certain value?

Reply:
> I agree with you that the keyboard recorder control requires a minimum width.
Maybe we could control the minimum width in the override public var intrinsicContentSize: CGSize { ... } function by not allowing a width of the size below a certain value?

👍